packages/gatsby-plugin-robots-txt/README.md

```diff
@@ -0,0 +1,36 @@
+# gatsby-plugin-robots-txt
+
+Provides support for writing a `robots.txt` file alongside your build with
+[robots-generator](https://www.npmjs.com/package/robots-generator).
+
+React Helmet is a component which lets you control your document head using
```

Comment:
Is this leftover from boilerplate?

Reply:
Welp good lookin out. Copied the README over. I'll kill that rn.